pandas/tests/test_resample.py

```diff
@@ -696,6 +696,32 @@ def create_index(self, *args, **kwargs):
         factory = self._index_factory()
         return factory(*args, **kwargs)
 
+    _index_fixture_start = datetime(2005, 1, 1)
+    _index_fixture_end = datetime(2005, 1, 10)
+    _index_fixture_freq = 'D'
+
+    @pytest.fixture(scope='class')
```

Comment:
normally don't use scoping like this (and instead use default function scoping). The issue is that we *could* have mutating things which will then propogate (we shouldn't in any event).